src/Raven.Client/ServerWide/Operations/DocumentsCompression/UpdateDocumentsCompressionConfigurationOperation.cs

```diff
@@ -10,14 +10,23 @@
 
 namespace Raven.Client.ServerWide.Operations.DocumentsCompression
 {
+    /// <summary>
+    /// Operation to update the documents compression configuration.
+    /// </summary>
     public sealed class UpdateDocumentsCompressionConfigurationOperation : IMaintenanceOperation<DocumentCompressionConfigurationResult>
     {
         private readonly DocumentsCompressionConfiguration _documentsCompressionConfiguration;
 
+        /// <inheritdoc cref="UpdateDocumentsCompressionConfigurationOperation"/>
+        /// <param name="configuration">
+        /// The <see cref="DocumentsCompressionConfiguration"/> object containing the new compression settings to apply.
```

Comment:
need to document `DocumentsCompressionConfiguration`

Reply:
done